pkg/manifests/types.go

```diff
@@ -753,6 +753,10 @@ type RemoteWriteSpec struct {
 	QueueConfig *monv1.QueueConfig `json:"queueConfig,omitempty"`
 	// Defines the timeout value for requests to the remote write endpoint.
 	RemoteTimeout string `json:"remoteTimeout,omitempty"`
+	// Enables sending of exemplars over remote write. Note that
+	// exemplar-storage itself must be enabled using the `spec.enableFeature`
```

Comment:
This should be managed automatically by CMO: if one remote write config enables exemplars then CMO should turn on the feature in Prometheus.